src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/LiquidViewFilters.cs

```diff
@@ -13,14 +14,21 @@ namespace OrchardCore.DisplayManagement.Liquid.Filters
 {
     public static class LiquidViewFilters
     {
+        private static readonly AsyncFilterDelegate _localizeDelegate = Localize;
+        private static readonly AsyncFilterDelegate _htmlClassDelegate = HtmlClass;
+        private static readonly AsyncFilterDelegate _newShapeDelegate = NewShape;
+        private static readonly AsyncFilterDelegate _shapeRenderDelegate = ShapeRender;
+        private static readonly AsyncFilterDelegate _shapeStringifyDelegate = ShapeStringify;
+        private static readonly FilterDelegate _shapePropertiesDelegate = ShapeProperties;
+
         public static FilterCollection WithLiquidViewFilters(this FilterCollection filters)
         {
-            filters.AddAsyncFilter("t", Localize);
-            filters.AddAsyncFilter("html_class", HtmlClass);
-            filters.AddAsyncFilter("shape_new", NewShape);
-            filters.AddAsyncFilter("shape_render", ShapeRender);
-            filters.AddAsyncFilter("shape_stringify", ShapeStringify);
-            filters.AddFilter("shape_properties", ShapeProperties);
+            filters.AddAsyncFilter("t", _localizeDelegate);
```

Comment:
Adding method reference boxes a delegate for calls, as this is called a lot doesn't make sense to create every time.

Reply:
Is it still true even if `WithLiquidViewFilters` is called once by the static ctor of `LiquidViewTemplate`?

Reply:
At least my debugger showed that this method was invoked for each page request? If this were once in static constructor kind of action, would not make sense to create this extra code.